spec.bs

```diff
@@ -1773,53 +1777,64 @@ null |winningComponentConfig|:
+    0 if encountered a null).
   1. If |sellerCurrency| is null:
-    1. [=map/Set=] |browserSignals|["`highestScoringOtherBid`"] to |leadingBidInfo|'s
+    1. Set |highestScoringOtherBid| to |leadingBidInfo|'s
       [=leading bid info/highest scoring other bid=]'s [=generated bid/bid=]'s
       [=bid with currency/value=] (or 0 if encountered a null).
-    1. [=map/Set=] |browserSignals|["`highestScoringOtherBidCurrency`"] to "`???`".
-  1. Otherwise:
-    1. [=map/Set=] |browserSignals|["`highestScoringOtherBid`"] to |leadingBidInfo|'s
-      [=leading bid info/highest scoring other bid=]'s [=generated bid/bid in seller currency=] (or
-      0 if encountered a null).
-    1. [=map/Set=] |browserSignals|["`highestScoringOtherBidCurrency`"] to |sellerCurrency|.
-  1. If |leadingBidInfo|'s [=leading bid info/scoring data version=] is not null, [=map/Set=]
-    |browserSignals|["`dataVersion`"] to it.
-  1. If |leadingBidInfo|'s [=leading bid info/top level seller=] is not null, [=map/set=]
-    |browserSignals|["`topLevelSeller`"] to it.
-  1. If |leadingBidInfo|'s [=leading bid info/top level seller signals=] is not null, [=map/set=]
-    |browserSignals|["`topLevelSellerSignals`"] to it.
-  1. If |leadingBidInfo|'s [=leading bid info/component seller=] is not null:
-    1. [=map/Set=] |browserSignals|["`componentSeller`"] to |leadingBidInfo|'s
-      [=leading bid info/component seller=].
-    1. If |winner|'s [=generated bid/modified bid=] is not null, [=map/set=]
-      |browserSignals|["`bid`"] to it.
-  1. Otherwise (if |leadingBidInfo|'s [=leading bid info/component seller=] is null), if |winner|'s
-    [=generated bid/modified bid=] is not null, [=map/set=] |browserSignals|["`modifiedBid`"] to it.
-  1. Set |config| to |leadingBidInfo|'s [=leading bid info/auction config=].
+  1. Let |bid| be |winner|'s [=generated bid/bid=]'s [=bid with currency/value=].
+  1. Let |modifiedBid| be null.
+  1. If |winner|'s [=generated bid/modified bid=] is not null:
+    1. If |leadingBidInfo|'s [=leading bid info/component seller=] is not null:
+      1. Set |bid| to |winner|'s [=generated bid/modified bid=].
+    1. Otherwise:
+      1. Set |modifiedBid| to |winner|'s [=generated bid/modified bid=].
+  1. Let |browserSignals| be a {{ReportResultBrowserSignals}} with the following fields:
+    <dl>
+      <dt>{{ReportingBrowserSignals/topWindowHostname}}
+      <dd>The result of running the <a spec=url>host serializer</a> on |[=this=]'s
+        [=relevant settings object=]'s [=environment/top-level origin=]'s [=origin/host=].
+      <dt>{{ReportingBrowserSignals/interestGroupOwner}}
+      <dd>[=serialization of an origin|Serialized=] |winner|'s [=generated bid/interest group=]'s
+        [=interest group/owner=].
+      <dt>{{ReportingBrowserSignals/renderURL}}
+      <dd>[=URL serializer|Serialized=] |winner|'s [=generated bid/ad descriptor=]'s [=ad descriptor/url=]

```

Comment:
Phrases like this should be "The serialization of X" or "The result of running the URL serializer on X", or "X, stochastically rounded." Ideally it would be clear to readers what's going to happen without needing to click through to a definition, so the serializations that go to URL or HTML can use short terms, but things defined in this spec should use longer ones. I'm not going to be picky about the details.

Reply:
done.